scripts/run_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+echo "Waiting for MySQL to start."
+attempts=0
+while ! nc -w 1 mysql 3306 | grep -q "mysql"; do
+  sleep 1
```

Comment:
Were you aiming for 60 seconds? If so, you can drop this sleep, since the `-w 1` act almost as a sleep (while mysql isn't running).

Reply:
Done!

Reply:
Seems like it would not sleep with `-w 1` and would error out after 60 continuous attempts. Added it back.
```
semian_1     | Waiting for MySQL to start.
.
.
.
.
semian_1     | mysql [172.18.0.3] 3306 (mysql) : Connection refused
semian_1     | ERROR: mysql was not started.
```

Reply:
Interesting. It must be able to fail the connection right away, instead of hang on waiting for it to establish :+1: